assets/uidraw.js

```
@@ -48,15 +48,75 @@ class UIDrawView {
+                definition(...this.params);
+                rendercontext = this.oldrc;
+            };
+
+            Component.prototype.bindstate = function (statevar) {
+                this.bindedstateval = statevar.content;
+                statevar.bindelement(this);
+                this.render();
+            };
+        }
+
+        return Component;
```

Biome (1.9.4)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor is ignored.

(lint/correctness/noConstructorReturn)